framework/docs/src/3_framework/8_ibc.md

```diff
@@ -1,24 +1,42 @@
 # Interchain Abstract Accounts
 
-The cosmos is going interchain. The arrival of Inter-Blockchain Communication (IBC) launched us into a new paradigm in application development. New use-cases and solutions to existing UX problems are now possible. As the interchain application platform, Abstract attempts to make IBC accessible to developers and users alike. In this section we'll delve into interchain application development, what makes it so hard and how Abstract can help you create your first #Interchain application.
+The cosmos is going interchain. The arrival of Inter-Blockchain Communication (IBC) launched us into a new paradigm in
+application development. New use-cases and solutions to existing UX problems are now possible. As the interchain
+application platform, Abstract attempts to make IBC accessible to developers and users alike. In this section we'll
+delve into interchain application development, what makes it so hard and how Abstract can help you create your first
+#Interchain application.
 
 ## What is The Interchain?
 
-The interchain, sometimes called the internet of blockchains, is the concept of a network of blockchains that can communicate with each other. This is made possible by the Inter-Blockchain Communication (IBC) protocol, which is a standard for communication between blockchains. IBC is a protocol that allows for the transfer of arbitrary data between blockchains. This means that users can send data (like tokens) from one blockchain to another without third-party trust assumptions. This is a huge step forward for the blockchain industry, as it allows for the creation of a network of blockchains that can independently communicate with each other.
+The interchain, sometimes called the internet of blockchains, is the concept of a network of blockchains that can
+communicate with each other. This is made possible by the Inter-Blockchain Communication (IBC) protocol, which is a
+standard for communication between blockchains. IBC is a protocol that allows for the transfer of arbitrary data between
+blockchains. This means that users can send data (like tokens) from one blockchain to another without third-party trust
+assumptions. This is a huge step forward for the blockchain industry, as it allows for the creation of a network of
+blockchains that can independently communicate with each other.
 
-```admonish info
-If you're interested in a visual representation of the interchain, check out [the map of zones](https://mapofzones.com/home?columnKey=ibcVolume&period=24h).
-```
+> If you're interested in a visual representation of the interchain, check
+> out <a href="https://mapofzones.com/home?columnKey=ibcVolume&period=24h" target="_blank">the map of zones</a>.
```

Comment:
I set every link that takes you out of the docs to open on a new tab. That's so that the reader is not taken away from where it left off.